app/ui/src/app/connections/list/list.component.html

```diff
@@ -108,24 +95,14 @@ <h1>Register Applications</h1>
         </div>
       </ng-container>
       <ng-container *ngIf="showNewConnection && connections.length > 0">
-        <div class="connection new-connection col-xs-12 col-sm-3 col-md-2 card"
-             (click)="onSelect(undefined, $event)">
-          <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
-            <div class="card-pf-body">
-
-              <!-- Card Icon -->
-              <div class="card-pf-top-element">
-                <span class="card-pf-icon-large pficon pficon-add-circle-o">
-                </span>
-              </div>
-
-              <!-- Card Description / Item Overview -->
-              <div class="card-pf-info text-center">
-                <button class="btn btn-default" (click)="onSelect(undefined, $event)">Create Connection</button>
-              </div>
-            </div>
+        <syndesis-empty-state-card class="col-md-6 col-lg-3" (click)="onSelect(undefined, $event)">
+          <div class="empty-state-card__header">
+            <span class="card-pf-icon-large pficon pficon-add-circle-o"></span>
```

Comment:
Do you believe the empty state card will always feature this large plus icon? If so, then I'd suggest to move it to the empty card element template instead. Otherwise this is perfectly fine.

Reply:
@deeleman, yeah I think it's safe to say that the large plus icon will always be there.